From a2f20fb2ea5c4a9d2410615686ab908972d514f5 Mon Sep 17 00:00:00 2001
From: Hai Pham <hai.pham.ud@renesas.com>
Date: Mon, 26 May 2025 20:50:09 +0700
Subject: [PATCH 40/52] arm64: dts: renesas: ironhide: Enable Ethernet PCS

Enable the Ethernet PCS and connect it to Multi-Protocol PHY

Signed-off-by: Thanh Quan <thanh.quan.xn@renesas.com>
Signed-off-by: Tam Nguyen <tam.nguyen.xa@renesas.com>
Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 arch/arm/dts/r8a78000-ironhide.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/dts/r8a78000-ironhide.dts b/arch/arm/dts/r8a78000-ironhide.dts
index 17f38f255fb..7a554f83adb 100644
--- a/arch/arm/dts/r8a78000-ironhide.dts
+++ b/arch/arm/dts/r8a78000-ironhide.dts
@@ -114,6 +114,11 @@
 	clock-frequency = <400000>;
 };
 
+&eth_pcs {
+	phys = <&mp_phy 2 1>;
+	status = "okay";
+};
+
 &mp_phy {
 	status = "okay";
 };
-- 
2.34.1

